mantidproject/mantid
# Move test guides into dev docs

**Description of work.**
Moves multiple test guides into the development docs from the user
documentation, as they are not intended for end-users

**To test:**
Check that both the developer and user documentation still build and that locally they move across (you may need to delete your `doctree` folder in your build directory)

*There is no associated issue.*

*This does not require release notes* because it is a documentation only change

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
